src/MessageImage.js

```diff
@@ -3,16 +3,27 @@ import {
   Image,
   StyleSheet,
   View,
+  Dimensions,
 } from 'react-native';
+import Lightbox from 'react-native-lightbox';
 
 export default class MessageImage extends React.Component {
   render() {
+    const { width, height } = Dimensions.get('window');
+
     return (
       <View style={[styles.container, this.props.containerStyle]}>
-        <Image
-          style={[styles.image, this.props.imageStyle]}
-          source={{uri: this.props.currentMessage.image}}
-        />
+        <Lightbox
+          navigator={this.props.navigator}
```

Comment:
I just added it to `propTypes` where it defaults to `null`. Hopefully this helps.